docs/core/install/linux-ubuntu.md

```diff
@@ -0,0 +1,336 @@
+| 2.2               | `dotnet-runtime-22` |
+| 2.1 (LTS)         | `dotnet-runtime-21` |
+
+Use the `snap install` command to install a .NET Core Runtime snap package. In this example, .NET Core 3.1 is installed:
+
+```bash
+sudo snap install dotnet-runtime-31 --classic
+```
+
+Next, register the `dotnet` command for the system with the `snap alias` command:
+
+```bash
+sudo snap alias dotnet-runtime-31.dotnet dotnet
+```
+
+This command is formatted as: `sudo snap alias {package}.{command} {alias}`. You can choose any `{alias}` name you would like. For example, you could name the command after the specific version installed by snap: `sudo snap alias dotnet-runtime-31.dotnet dotnet31`. When you use the command `dotnet31`, you'll invoke this specific version of .NET. But this is incompatible with most tutorials and examples as they expect a `dotnet` command to be available.
+
+## Scripted install
+
+The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for automation and non-admin installs of the **SDK**. You can download the script from <https://dot.net/v1/dotnet-install.sh>.
+
+The script defaults to installing the latest [long term support (LTS)](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) version, which is .NET Core 3.1. To install the current release, which may not be an (LTS) version, use the `-c Current` parameter.
+
+```bash
+./dotnet-install.sh -c Current
+```
+
+To install .NET Core Runtime instead of the SDK, use the `--runtime` parameter.
+
+```bash
+./dotnet-install.sh -c Current --runtime
+```
+
+You can install a specific version by altering the `-c` parameter to indicate the specific version. The following command installs .NET Core SDK 3.0.
+
+```bash
+./dotnet-install.sh -c 3.0
+```
+
+For more information, see [dotnet-install scripts reference](../tools/dotnet-install-script.md).
+
+## Manual install
+
+Both .NET Core SDK and .NET Core Runtime can be manually installed after they've been downloaded. First, download a binary release for either the SDK or the runtime from one of the following sites:
+
+- [.NET 5.0 preview downloads](https://dotnet.microsoft.com/download/dotnet/5.0)
+- [.NET Core 3.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0)
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2)
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+
+Next, extract the downloaded file and use the `export` command to set variables used by .NET Core and then ensure .NET Core is in PATH.
+
+To extract the runtime and make the .NET Core CLI commands available at the terminal, first download a .NET Core binary release. Then, open a terminal and run the following commands from the directory where the file was saved.
+
+```bash
+mkdir -p $HOME/dotnet && tar zxf aspnetcore-runtime-3.1.0-linux-x64.tar.gz -C $HOME/dotnet
```

Comment:
Consider quoting to handle spaces:

```suggestion
mkdir -p "$HOME/dotnet" && tar zxf aspnetcore-runtime-3.1.0-linux-x64.tar.gz -C "$HOME/dotnet"
```

(Not necessary for the exports, just the command.)